pkg/volume/glusterfs/glusterfs.go

```diff
@@ -221,6 +222,13 @@ func (b *glusterfsBuilder) setUpAtInternal(dir string) error {
 		options = append(options, "ro")
 	}
 
+	p := path.Join(b.glusterfs.plugin.host.GetPluginDir(glusterfsPluginName), b.glusterfs.volName)
+	if err := os.MkdirAll(p, 0750); err != nil {
+		return err
```

Comment:
this is rare, but I prefer not exiting so early. How about if `MkdirAll` succeeds, then append `log-file` option.